test/test_sso.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, username=TESTINGNAME, email=DEFAULTEMAIL, sub=TESTINGSUB, gro
     def authorize_access_token(self):
         if self.throws:
             raise self.throws
-        return "Schinken"
+        return {'userinfo': self.parse_id_token(None)}
 
     def parse_id_token(self, token):
```

Comment:
The goal of this mock function was to be in line with the signature of the upstream api. As we do not call `parse_id_token` ourselves now, we could remove the `token` part from the signature -- However in the context of this actually being a mock of the upstream API, we should probably instead adapt to `token, nonce, claims_options=None, leeway=120` to make this an actually API compatible mocked function.